hover.sh

```diff
@@ -3,20 +3,48 @@ export HOVER_OVERLAY_DIR="$(mktemp -d)"
 mkdir $HOVER_OVERLAY_DIR/{home,.upper,.work}
 export HOVER_HOME=$HOVER_OVERLAY_DIR/home
 
+show_help() {
+  local b=$(tput bold)
```

Comment:
`tput` comes from the `ncurses` package, please add that to the `runtimeInputs` list in the flake.nix.